third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -65,6 +65,29 @@ func TestAccBigQueryTable_Kms(t *testing.T) {
 	})
 }
 
+func TestAccBigQueryTable_RangePartitioning(t *testing.T) {
```

Comment:
please also version-gate these test functions :)

Reply:
How can I do that? I did not find any examples.

Reply:
the same way lol...

Reply:
oh, the test file is a Golang file - we'll have to

a) rename this file to resource_bigquery_table_test.go.erb
b) add the gates (`<% unless version == 'ga' -%>`)